src/site/content/en/blog/idle-detection/index.md

```diff
@@ -180,7 +153,7 @@ console.log('IdleDetector is stopped.');
 
 ### DevTools support
 
-Starting in Chrome&nbsp;86, you can emulate idle events in Chrome DevTools without actually being idle.
+Starting in Chromium&nbsp;86, you can emulate idle events in DevTools without actually being idle.
```

Comment:
Should you update to the lauch version, rather than the original flagged versoin?

Reply:
Well, it was available as early as 86, so I think it's fine to leave it as is.

Reply:
Because it was an origin trial, it's not usable in anything before 94. Please change.

Reply:
Changed then.